### DISCUSSION TITLE
Episode status is "unset

### DISCUSSION THREAD

**Discussion by mrmischeif1** [pymedusa/Medusa#10402] [Q&A]
hi

can anyone please assist me with episodes periodically changing to the status with unset.

please refer to picture of example of one of the multiple shows this happens to overtime

previous episodes have been snatched, 
![Capture](https://user-images.githubusercontent.com/73575228/157003475-83ad30ee-5b28-4ef4-ad51-878c888b8c21.JPG)
with the screenshot showing upcoming episodes being "unset"

**Comment by p0psicles** [comment]
No idea. So they started with a different status? Or they where unset initially?

**Reply by mrmischeif1** [reply]
the initial status i assume was "not aired", certainly not unset
